### PR TITLE
[misc]: update format workflow

### PR DESCRIPTION
**Pull request by 0x009922**
Signed-off-by: 0x009922 <a.marcius26@gmail.com>

Changes:

- Remove format-and-push workflow. It doesn't work if PR is made from a fork, which is the most common case for this repo.
- Add format check workflow. If there are formatting issues, it will fail and print "invalid" files in logs.
- Format existing files; fix formatting script that skipped them before
- Update contribution guide in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,50 +2,61 @@
...
-```
+3. **Install packages:**
 
-That's it!
+   ```bash
+   pnpm i
+   ```
 
-### Local dev-hot-reload mode
+Then you can use
```
Comment (outoftardis) [inline README.md:38]: then you can use what?

Comment (0x009922) [reply README.md:38]: Good question

File: README.md [hunk 2]
```diff
@@ -2,51 +2,62 @@
...
 
-### Format Markdown
+### Documentation formatting
 
-```bash
-pnpm format:docs
-```
+We use [Prettier](https://prettier.io/) to format Markdown files. Its configuration is located at `./.prettierrc.js`. For available options please follow to [Options reference](https://prettier.io/docs/en/options.html).
+
+- **Format doc files** - apply Prettier's formatting to all Markdown files
```
Comment (outoftardis) [inline README.md:50]:
```suggestion
- **Format doc files**: apply `Prettier` formatting to all Markdown files
```

File: README.md [hunk 3]
```diff
@@ -2,51 +2,62 @@
...
-This will start a local dev-server so you can open a browser and observe documentation and see your edits on-demand.
+It will start a local dev-server. You will be able to open a browser, observe rendered documentation, edit source files and see your edits on-demand.
 
-### Format Markdown
+### Documentation formatting
 
-```bash
-pnpm format:docs
-```
+We use [Prettier](https://prettier.io/) to format Markdown files. Its configuration is located at `./.prettierrc.js`. For available options please follow to [Options reference](https://prettier.io/docs/en/options.html).
```
Comment (outoftardis) [inline README.md:48]:
```suggestion
We use [Prettier](https://prettier.io/) to format Markdown files. Its configuration is located at `./.prettierrc.js`. Check [options reference](https://prettier.io/docs/en/options.html) for all available options.
```

**Review by outoftardis — APPROVED** [review]

**Pull request merged**